server/src/main/java/org/opensearch/http/DefaultRestChannel.java

```diff
@@ -69,13 +71,16 @@ public class DefaultRestChannel extends AbstractRestChannel implements RestChann
     static final String CONTENT_TYPE = "content-type";
     static final String CONTENT_LENGTH = "content-length";
     static final String SET_COOKIE = "set-cookie";
+    static final String SERVER_VERSION = "server-version";
```

Comment:
```suggestion
    static final String SERVER_VERSION = "X-Server-Version";
```

or 

```suggestion
    static final String SERVER_VERSION = "X-OpenSearch-Version";
```

Reply:
Why is the X needed?

Reply:
This is a 100% custom header [1] which will never become standards, see please usage of `X-Opaque-ID` for example

[1] https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers

Reply:
Ahhh didnt see the previous comment this has been refactored